cherab/openadas/install.py

```diff
@@ -104,6 +107,26 @@ def install_adf11acd(element, file_path, download=False, repository_path=None, a
     repository.update_recombination_rates(rate, repository_path)
 
 
+def install_adf11ccd(element, file_path, download=False, repository_path=None, adas_path=None):
+    """
+    Adds the thermal charge exchange rate defined in an ADF11 file to the repository.
+
+    :param element: The element described by the rate file.
```

Comment:
Isn't this the receiving element? In other parts of the code we specify them as `donor_ion` and `receiver_ion`. This makes it very clear which are the two bodies in the interaction and which role they play. Have a look at some of the call signatures for other rates, like this one. Even if OpenADAS currently only has data for hydrogen donors, the API should specify both allowing for more data to be added in future.

https://github.com/cherab/openadas/blob/8a61dce0dfb2634ed35cf1275b848474ee631c7a/cherab/openadas/install.py#L49

Reply:
The RecursiveDict structure is being formed inside the [adf12  parsing funciton](https://github.com/Mateasek/cherab_openadas/blob/3fc091e22d6e855ec86207730ff927ad338c814a/cherab/openadas/parse/adf12.py#L47). For ADF11 it would be kind of redundant for all the file types except ccd. I will add construction of the requested tree to the [install.py](https://github.com/Mateasek/cherab_openadas/blob/3fc091e22d6e855ec86207730ff927ad338c814a/cherab/openadas/install.py#L110).